cmd/plan.go

```diff
@@ -68,6 +68,11 @@ func plan(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+// GetInstalledPackages returns a list of PriorInstalledPackages
+func GetInstalledPackages() map[string]desc.Desc {
```

Comment:
why have this, rather than just calling pacman.GetPriorInstalledPackages directly?

Reply:
I felt it was cleaner to not pull pacman into Install.go.  Install.go feels like a top-level function that should call well-defined public functions - suitable for consumption without much context or inner knowledge of the package.  that _was_ my opinion, I now see that Install.go imports gpsr, so it's probably better to import pacman and call directly